packages/typespec-python/test/azure/generated/payload-multipart/payload/multipart/_vendor.py

```diff
@@ -24,12 +25,20 @@ def __init__(self, name: str, *args, **kwargs):
         self.name = name
 
 
-def multipart_form_data_file(file: Union[IOBase, bytes]) -> IOBase:
+def multipart_file(file: Union[IOBase, bytes]) -> IOBase:
     if isinstance(file, IOBase):
         return file
     return NamedBytesIO("auto-name-" + str(uuid.uuid4()), file)
 
 
+def multipart_data(data: Any) -> Any:
+    serialized_data = _serialize(data)
+
+    if isinstance(serialized_data, (list, tuple, dict)):
+        return json.dumps(serialized_data, cls=SdkJSONEncoder, exclude_readonly=True)
+    return serialized_data
```

Comment:
You could use `json.dumps` directly I think.

Reply:
make sense